__tests__/books.test.js

```diff
@@ -0,0 +1,48 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('books routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('/api/v1/books/ should return a list of books', async () => {
+    const resp = await request(app).get('/api/v1/books');
+    expect(resp.status).toBe(200);
+    expect(resp.body).toEqual([
+      { id: '1', title: 'The Old Man and the Sea', released: '1952' },
+      { id: '2', title: 'A Farewell to Arms', released: '1929' },
+      { id: '3', title: 'Twilight', released: '2005' },
+      { id: '4', title: 'It', released: '1986' },
+      { id: '5', title: 'The Shining', released: '1977' },
+      { id: '6', title: 'The Lightning Thief', released: '2005' },
+      { id: '7', title: 'The Sea of Monsters', released: '2006' },
+    ]);
```

Comment:
Here I like to use expect.arrayContaining(...) within my .toEqual() as any further data that is added will cause the tests to fail in the future.